lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -1120,7 +1097,56 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
             let epochs = poolGarbageCollectionEpochNo <$> events
             (reverse epochs `zip` [1 ..]) `shouldSatisfy` all (uncurry (==))
 
+    it "STAKE_POOLS_SMASH_01 - fetching metadata from SMASH works with delisted pools" $
+        \ctx -> runResourceT $ do
+            smashUrl <- liftIO $ getEnv "CARDANO_WALLET_SMASH_URL"
+            updateMetadataSource ctx (T.pack smashUrl)
```

Comment:
We should run the test in a form of "bracket-style resource handler", which set the metadataSource back to what it was after the test. Otherwise it may have an influence on other tests depending on which one runs first.